### PR TITLE
New tool: recording splitter

### PR DESCRIPTION
**Pull request by ticcky**
This PR contains a new tool for splitting PCM/WAV recordings into individual WAVs and it also generates `session.xml` files needed for the transcription tool.

For smooth incorporation into the existing codebase I had to generalize the VAD component.

### CODE REVIEW

File: alex/corpustools/recording_splitter/autopath.py [hunk 1]
```diff
@@ -0,0 +1,132 @@
+"""
```
Comment (jurcicek) [inline alex/corpustools/recording_splitter/autopath.py]: dej do gity sim link, a ne orig soubor.

File: alex/corpustools/recording_splitter/recording_splitter.py [hunk 2]
```diff
@@ -0,0 +1,356 @@
+from collections import deque
```
Comment (jurcicek) [inline alex/corpustools/recording_splitter/recording_splitter.py]: do not put it into one one directory as a unique file. just put it into the corpustools

**Comment by jurcicek** [comment]
Did you tested the changes in VAD with a voip system by calling it?

File: alex/corpustools/recording_splitter/recording_splitter.py (hunk 2 above)
Comment (ticcky) [inline alex/corpustools/recording_splitter/recording_splitter.py]: Done.

**Comment by ticcky** [comment]
I fixed a small bug and made the changes you requested.

**Comment by jurcicek** [comment]
@ticcky Did you tested the changes in VAD with a voip system (e.g. PTIcs) by calling it?

**Comment by ticcky** [comment]
Yes I did, sorry I did not confirm.

**Pull request merged**